vispy/visuals/graphs/layouts/__init__.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+
+from .random import random
+from .circular import circular
+from .force_directed import fruchterman_reingold
+
+
+_layout_map = {
+    'random': random,
+    'circular': circular,
+    'force_directed': fruchterman_reingold,
+    'spring_layout': fruchterman_reingold
+}
+
+AVAILABLE_LAYOUTS = _layout_map.keys()
+
+
+def get(name, *args, **kwargs):
```

Comment:
This should at least be `get_layout`. Can we make it private? Or do you think users will need this?